src/test/resources/logback-test.xml

```diff
@@ -15,8 +15,18 @@
         </layout>
     </appender>
 
-    <root level="ERROR">
+    <appender name="FILE" class="ch.qos.logback.core.FileAppender">
+        <File>${log-path}/beamLog.out</File>
+        <layout class="ch.qos.logback.classic.PatternLayout">
+            <Pattern>
+                %d{HH:mm:ss.SSS} [%thread] %-5level %logger{36} - %msg%n
+            </Pattern>
+        </layout>
+    </appender>
+
+    <root level="INFO">
```

Comment:
beamLog.out should be info, but console should be Error level

Reply:
done